Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/RecordNavPage.xaml

```diff
@@ -6,31 +6,39 @@
       xmlns:userControl="clr-namespace:Ginger.UserControlsLib"
       xmlns:usercontrols="clr-namespace:Amdocs.Ginger.UserControls"
       xmlns:local="clr-namespace:Ginger.BusinessFlowsLibNew.AddActionMenu"
+      xmlns:Ginger="clr-namespace:Ginger"
       mc:Ignorable="d" 
-      Title="RecordNavAction">
+      Title="RecordNavAction" >
 
     <Grid>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="300*"/>
+            <ColumnDefinition Width="400*"/>
         </Grid.ColumnDefinitions>
         <Grid.RowDefinitions>
-            <RowDefinition Height="30" />
+            <RowDefinition Height="45" />
+            <RowDefinition Height="40" />
+            <RowDefinition Height="40" />
             <RowDefinition Height="Auto" />
-        </Grid.RowDefinitions>
+        </Grid.RowDefinitions>        
+        
+        <userControl:UCWindowsGrid x:Name="xWinGridUC" Margin="0, 5, 0, 0" Grid.Row="0" AddSwitchActionRequired="True"  />
 
-        <userControl:UCWindowsGrid x:Name="xWinGridUC" Grid.Row="0" AddSwitchActionRequired="False" />
+        <usercontrols:ucButton Grid.Row="1" x:Name="xStartAgentButton" Margin="3,0,0,0" ButtonType="RoundTextAndImageButton" ButtonText="Start Agent" ButtonImageType="Agent" 
```

Comment:
Start Agent button should not be here, page should trigger Agent start automatically only if it is needed due to user operations like StartRecording.

Reply:
it will be difficult if we are starting the agent on start recording, suppose if the agent is started on start recording then it will open the browser first then user will put the url to browse after that we will required the script injection so it cannot be done in 1 button click, please let me know if still need to go in this direction.